utf8proc.c

```diff
@@ -186,6 +186,37 @@ UTF8PROC_DLLEXPORT utf8proc_ssize_t utf8proc_encode_char(utf8proc_int32_t uc, ut
 }
 
 /* internal "unsafe" version that does not check whether uc is in range */
```

Comment:
OK, that came about because I first did the changes based on what I found in my julia directory... only later did I see that the code really lived in JuliaLang/utf8proc instead of JuliaLang/julia, and missed fixing those... Sorry!